common/app/views/fragments/header.scala.html

```diff
@@ -118,6 +118,9 @@
                                 <span class="inline-logo inline-guardian-logo-320"></span>
                             <![endif]-->
                         }
+                        <object class="hide-on-mobile hide-on-slim-header">
```

Comment:
enables you to embed an `<a>` within an `<a>`. handily, i'd just [read about it](https://speakerdeck.com/smashingmag/dirty-tricks-from-the-dark-corners-of-front-end?slide=8) in @smashingmagazine presentation